doc/Developing-Step-By-Step-Angular.html

```diff
@@ -186,12 +186,12 @@ <h3>Creating Person Entity</h3>
 <p>We define entities in <strong>.Core</strong> (domain) project (in server 
 side). We can define a <strong>Person</strong> entity (mapped to <strong>PbPersons</strong> 
 table in database) to represent a person in phone book as shown below (I created 
-in in a new folder/namespace named People):</p>
+in in a new folder/namespace named PhoneBook):</p>
 <pre lang="cs">using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 using Abp.Domain.Entities.Auditing;
 
-namespace Acme.PhoneBook<span lang="tr">Demo</span>.People
+namespace Acme.PhoneBook<span lang="tr">Demo</span>.<span lang="tr">P<span class="auto-style1">honeBook</span></span>
```

Comment:
There's a lot of `<span lang="tr">` that seem unnecessary. And `<span class="auto-style1">` too.

Reply:
Thank you @acjh.
@alirizaadiyahsi can you please fix them all. We don't like mess HTML :)